.github/workflows/scorecard.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         id: checksecret
         uses: ./.github/actions/is-defined
         with:
-          value: ${{ secrets.SCORECARD_READ_TOKEN }}
+          value: ${{ secrets.SCORECARD_TOKEN }}
```

Comment:
why did you change the secret name ?

Reply:
Ah good point, updated `SCORECARD_READ_TOKEN`.